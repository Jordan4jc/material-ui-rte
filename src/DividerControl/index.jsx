import React from "react"
import { ButtonGroup, Button } from "@material-ui/core"
import RemoveIcon from "@material-ui/icons/Remove"

const DividerControl = props => {
  return (
    <ButtonGroup variant="contained" size="small">
      <Button onClick={() => props.onClick()}>
        <RemoveIcon />
      </Button>
    </ButtonGroup>
  )
}

export default DividerControl
