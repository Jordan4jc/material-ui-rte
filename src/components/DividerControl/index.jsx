import React from "react"
import ButtonGroup from "@material-ui/core/ButtonGroup"
import Button from "@material-ui/core/Button"
import RemoveIcon from "@material-ui/icons/Remove"

const DividerControl = props => {
  return (
    <ButtonGroup variant="contained" size="small">
      <Button onClick={() => props.onClick()}>
        <RemoveIcon />
      </Button>
    </ButtonGroup>
  )
}

export default DividerControl
