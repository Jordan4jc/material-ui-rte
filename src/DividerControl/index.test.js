import React from "react"
import { shallow } from "enzyme"
import DividerControl from "./index"

describe("DividerControl", () => {

  it("Should render alignment control", () => {
    const component = shallow(<DividerControl />);
  
    expect(component).toMatchSnapshot();
  });
});