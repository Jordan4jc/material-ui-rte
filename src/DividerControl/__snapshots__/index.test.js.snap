// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DividerControl Should render alignment control 1`] = `ShallowWrapper {}`;
